File: crates/deslop/tests/fixtures/dart-type3/delta.dart
int accumulate(int bound) {
  if (bound < 0) {
    return 0;
  }
  var running = 0;
  for (var step = 0; step < bound; step = step + 1) {
    running = running + step;
    running = running + 2;
  }
  return running;
}

File: crates/deslop/tests/fixtures/typescript-type3/delta.ts
export function accumulate(values: number[]): number {
  let running = 0;
  for (const step of values) {
    const scaled = step * 3;
    running = running + scaled;
    if (running > 50) {
      running = running - 5;
    }
  }
  running = running + 2;
  return running;
}
